frontend/components/GamesList.tsx

```diff
@@ -3,31 +3,45 @@ import { connect } from 'react-redux';
 import { AppStore } from '../storage/common';
 import { Game } from '../storage/models/Game';
 import { GameListRow } from './listings/GameListRow';
+import { GameSeries } from '../storage/models/GameSeries';
+import { File } from '../storage/models/File';
 interface GamesFilterProps {
 	filterString: string;
 }
 interface GamesListProps extends GamesFilterProps {
 	games: Array<Game>;
+	gameSeries: Map<number, GameSeries>;
+	images: Map<number, File>;
 	gamesVisibility: Map<number, boolean>;
 }
 
 const GamesList: React.FC<GamesListProps> = (props: GamesListProps) => {
 	const { games, gamesVisibility } = props;
-	const filteredGames = games.filter((game: Game) => game.name.startsWith(props.filterString));
+	const filteredGames = games.filter((game: Game) => game.title.startsWith(props.filterString));
 	return (
 		<>
 			{filteredGames.map(game => {
 				if (gamesVisibility.get(game.id) ?? true) {
-					return <GameListRow key={game.id} img={game.img} name={game.name} series={game.series} />;
+					let imgUrl = '';
+					let gameSeries = '';
```

Comment:
Should be either `gameSeriesName` or just `seriesName` since it holds the name, not the series itself